GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -266,6 +266,7 @@ public IReadOnlyList<HotkeySettings> CreateDefaultSettings()
                     Hk(FormBrowse.Command.StashPop, Keys.Control | Keys.Alt | Keys.Down),
                     Hk(FormBrowse.Command.StashStaged, Keys.Control | Keys.Shift | Keys.Alt | Keys.Up),
                     Hk(FormBrowse.Command.ToggleBetweenArtificialAndHeadCommits, Keys.Control | Keys.OemBackslash),
+                    Hk(FormBrowse.Command.ToggleAndFocusOutputHistory, Keys.Control | Keys.D9),
```

Comment:
Link to the behavior, maybe naming it `FormBrowse.Command.Toggle**Or**FocusOutputHistory` makes more sense...